src/emulator/storage/files.ts

```diff
@@ -120,7 +120,7 @@ export enum UploadStatus {
   FINISHED,
 }
 
-export type FinalizedUpload = {
+export type FinishedUpload = {
```

Comment:
JSDoc for all exported types as well

Reply:
Deleting this type. We'll just return the `StoredFile` since we're passing the `ResumableUpload`.